### PR TITLE
kw2xrf: short address getter and iid option

### PR DESCRIPTION
**Pull request by OlegHahm**
Without this gnrc IPv6 won't work on the Phytec board.

### CODE REVIEW

**Comment by authmillenon** [comment]
I have no idea of the device, but from the GNRC perspective you are right. For the rest I reassign.

**Comment by thomaseichinger** [comment]
I have no device to test right now, but the code looks good. Will test tomorrow.

File: drivers/kw2xrf/kw2xrf.c [hunk 1]
```diff
@@ -563,6 +569,22 @@ int kw2xrf_get(gnrc_netdev_t *netdev, netopt_t opt, void *value, size_t max_len)
...
+                ieee802154_get_iid(value, (uint8_t *)&addr, 8);
+            }
+            else {
+                uint16_t addr = kw2xrf_get_addr_short(dev);
+                ieee802154_get_iid(value, (uint8_t *)&addr, 2);
+            }
+            return sizeof(eui64_t);
+
+
+
```
Comment (thomaseichinger) [inline drivers/kw2xrf/kw2xrf.c]: Please remove extra newlines.

**Comment by thomaseichinger** [comment]
Works like a charm, ACK when comment is addressed.

**Comment by OlegHahm** [comment]
Removed superfluous newlines and squashed directly.

**Pull request merged**